src/xform_to_datalog/datalog_facts.h

```diff
@@ -66,13 +67,17 @@ class DatalogFacts {
 .decl duplicateTestCaseNames(testAspectName: symbol)
 .output duplicateTestCaseNames
 
+.output disallowedUsage
```

Comment:
Perhaps this should have `(IO=stdout)` to prevent the issue we addressed in #208 ?

Reply:
Right. fixed. Thanks again.